### DISCUSSION TITLE
@artilleryio/platform-fargate license

### DISCUSSION THREAD

**Discussion by SamuelSSalazar** [artilleryio/artillery#1970] [Artillery]
Hello Artillery team,
I am trying to use Artillery in my company to run load tests using AWS Fargate. However, in order to use it we need to have clarity on the license that applies to the `@artilleryio/platform-fargate` package. Does the [MPLv2](https://www.mozilla.org/en-US/MPL/2.0/) also applies to this package? I was not able to found the code/license for it in github and the legal department of my company has denied the usage of the platform because it seems that the `@artilleryio/platform-fargate` falls under the [Artillery commercial license](https://www.artillery.io/terms/eula).

**Comment by hassy** [comment]
Hi @SamuelSSalazar, thanks for flagging this up! It's an oversight on our part, we'll republish the package with the updated license field. It's meant to be MPL 2.0 like the rest of Artillery.

**Reply by SamuelSSalazar** [reply]
Thanks @hassy
